docs/book/src/images/cluster-admission-machinedeployment-controller.plantuml

```diff
@@ -31,6 +31,27 @@ repeat
       elseif (New MachineSets Replicas available) then (yes)
         #LightBlue:Scale MachineSet down;
       endif
+    elseif (OnDelete Deployment Strategy) then (yes)
```

Comment:
I don't see https://deploy-preview-4346--kubernetes-sigs-cluster-api.netlify.app/developer/architecture/controllers/machine-deployment.html being updated in the preview

Reply:
The png for the diagram will need to be updated by calling `make diagrams`

Reply:
done thank you both for the tips and great comment!